build/configs/artik053/artik053_download.sh

```diff
@@ -64,15 +70,33 @@ main()
 				exit 1
 			fi
 
+			# Make a romfs image
+			if [ "$ROMFS_SUPPORT" = "y" ]; then
+				pushd ${OS_DIR_PATH}
+					sh ../tools/fs/mkromfsimg.sh
+				popd
+			fi
+
 			# Download all binaries using openocd script
 			pushd ${OPENOCD_DIR_PATH}
-			${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
-				flash_write bl1 ../../bin/bl1.bin; 		\
-				flash_write bl2 ../../bin/bl2.bin; 		\
-				flash_write sssfw ../../bin/sssfw.bin; 		\
-				flash_write wlanfw ../../bin/wlanfw.bin;	\
-				flash_write os ../../../../output/bin/tinyara_head.bin;	\
-			exit'
+			if [ "$ROMFS_SUPPORT" = "y" ]; then
+				${OPENOCD_BIN_PATH}/openocd -f artik053_romfs.cfg -c ' 	\
+					flash_write bl1 ../../bin/bl1.bin; 		\
+					flash_write bl2 ../../bin/bl2.bin; 		\
+					flash_write sssfw ../../bin/sssfw.bin; 		\
+					flash_write wlanfw ../../bin/wlanfw.bin;	\
+					flash_write os ../../../../output/bin/tinyara_head.bin;	\
+					flash_write rom ../../../../output/bin/romfs.img;	\
```

Comment:
The difference is small - just a single argument is added. Please fold these two cases into one.

Reply:
Ok. Thank you.